metadata-ingestion/scripts/modeldocgen.py

```diff
@@ -478,10 +482,14 @@ def strip_types(field_path: str) -> str:
                     BrowsePathsClass([f"/prod/datahub/entities/{entity_display_name}"]),
                 ],
             )
+            dataset.browse_path_v2 = BrowsePathsV2Class(  # type: ignore
```

Comment:
I don't think `dataset.browse_path_v2` exists, and it's not part of the snapshot class. You'll need to emit a separate MCPW